src/codec.rs

```diff
@@ -57,13 +61,23 @@ pub mod pb_codec {
 
     #[inline]
     pub fn de<T: Message>(mut reader: MessageReader) -> Result<T> {
-        let mut s = CodedInputStream::from_buffered_reader(&mut reader);
+        let mut s = from_buf_read(&mut reader);
         let mut m = T::new();
         m.merge_from(&mut s)?;
         Ok(m)
     }
 }
 
+#[cfg(feature = "protobuf-codec")]
+fn from_buf_read(reader: & mut MessageReader) -> protobuf::CodedInputStream {
+    return protobuf::CodedInputStream::from_buffered_reader(reader);
```

Comment:
No need to return in Rust.

Reply:
Fixed overall through the PR (except in auto-generated files in case someone re-generates them). thanks.